rocAL/rocAL_pybind/example/coco_pipeline.py

```diff
@@ -160,131 +169,66 @@ def __next__(self):
             self.loader.copyToTensorNHWC(
                 self.out, self.multiplier, self.offset, self.reverse_channels, int(self.tensor_dtype))
 
-        self.img_names_length = np.empty(self.bs, dtype="int32")
-        self.img_names_size = self.loader.GetImageNameLen(
-            self.img_names_length)
-        print("Image name length:", self.img_names_size)
-# Images names of a batch
-        self.Img_name = self.loader.GetImageName(self.img_names_size)
-        print("Image names in a batch ", self.Img_name)
+#Image id of a batch of images
+        self.loader.GetImageId(self.image_id)
 # Count of labels/ bboxes in a batch
-        self.bboxes_label_count = np.zeros(self.bs, dtype="int32")
-        self.count_batch = self.loader.GetBoundingBoxCount(
-            self.bboxes_label_count)
+        self.count_batch = self.loader.GetBoundingBoxCount(self.bboxes_label_count)
         print("Count Batch:", self.count_batch)
-# 1D labels array in a batch
-        self.labels = np.zeros(self.count_batch, dtype="int32")
-        self.loader.GetBBLabels(self.labels)
-        print(self.labels)
-# 1D bboxes array in a batch
-        self.bboxes = np.zeros((self.count_batch*4), dtype="float32")
-        self.loader.GetBBCords(self.bboxes)
-        print(self.bboxes)
+# 1D labels & bboxes array
+        self.encoded_bboxes = np.zeros((self.count_batch*4), dtype="float32")
```

Comment:
move all allocation to init() function to avoid doing it every iteration

Reply:
In cases when we don't use the BoxEncoder in example files, the number of bounding boxes for every batch of images will be different.Hence kept this here.
In MLPerf, since we use the boxencoder, have moved it to the init() by avoiding it for every iteration